### PR TITLE
ci: Enable vbox recording on tests

### PR DESCRIPTION
**Pull request by Itxaka**
Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

**Comment by Itxaka** [comment]
Seems to work :D

File: packer/images.json.pkr.hcl [hunk 1]
```diff
@@ -151,6 +151,9 @@ source "virtualbox-iso" "cos" {
   ssh_timeout            = "5m"
   ssh_username           = "${var.root_username}"
   vm_name                = "cOS"
+  vboxmanage = [
+    ["modifyvm", "{{.Name}}", "--recording", "on", "--recordingscreens", "0","--recordingfile", "../capture.webm"],
```
Comment (mudler) [inline packer/images.json.pkr.hcl:155]: mmm just a silly question, is this going to record nevertheless when we ran the packer template? there is no way to optionally turn it off/on?

**Review by mudler — APPROVED** [review]
Looks good here!

**Comment by mudler** [comment]
mmm seeing this in test-raw disks:

```
Run actions/upload-artifact@v2
With the provided path, there will be 1 file uploaded
events.js:187
      throw er; // Unhandled 'error' event
      ^

Error: EACCES: permission denied, open '/Users/runner/work/cOS-toolkit/cOS-toolkit/capture.webm'
Emitted 'error' event on ReadStream instance at:
    at internal/fs/streams.js:120:12
    at FSReqCallback.oncomplete (fs.js:146:23) {
  errno: -13,
  code: 'EACCES',
  syscall: 'open',
  path: '/Users/runner/work/cOS-toolkit/cOS-toolkit/capture.webm'
}
```
https://github.com/rancher-sandbox/cOS-toolkit/runs/4211735613?check_suite_focus=true

File: packer/images.json.pkr.hcl (hunk 1 above)
Comment (Itxaka) [reply packer/images.json.pkr.hcl:155]: no way of turning off for now. Do we want to skip it?

Comment (Itxaka) [reply packer/images.json.pkr.hcl:155]: I mean, we could just not do it in the packer or hide it under a debug env var I guess, packer is very consistent and usually never fails, and its easy to reproduce locally

Comment (mudler) [reply packer/images.json.pkr.hcl:155]: :+1: fine by me, was just curious.. I just wondered about webm support. Is something backed inside Vbox I guess, so should be fine!

**Comment by Itxaka** [comment]
> mmm seeing this in test-raw disks:

should be fixed, permissions issue

File: packer/images.json.pkr.hcl (hunk 1 above)
Comment (Itxaka) [reply packer/images.json.pkr.hcl]: there, updated with a switch so its off by default

Comment (mudler) [reply packer/images.json.pkr.hcl]: Awesome! 👍

**Comment by Itxaka** [comment]
mmhh, this actually introduces a kind of heavy impact on the speed, with some tests taking up to 400 more seconds which I dont think its viable.

Im thinking Im gonna set it behind a debug keyword like on packer (capture_enabled) and have it disabled by default.

**Pull request merged**